mods/default/nodes.lua

```diff
@@ -1449,6 +1451,51 @@ minetest.register_node("default:acacia_bush_sapling", {
 	end,
 })
 
+minetest.register_node("default:sand_with_kelp", {
+	description = "Kelp On Sand",
+	drawtype = "plantlike_rooted",
+	tiles = {"default_sand.png"},
+	special_tiles = {{name = "default_kelp.png", tileable_vertical = true}},
+	inventory_image = "default_kelp.png",
+	paramtype2 = "leveled",
+	groups = {snappy = 3},
+	node_placement_prediction = "",
+
+	on_place = function(itemstack, placer, pointed_thing)
+		local node_ptu = minetest.get_node(pointed_thing.under)
+		local def_ptu = minetest.registered_nodes[node_ptu.name]
+		if def_ptu and def_ptu.on_rightclick then
```

Comment:
shouldn't shift override the pointed at node's right click? To allow you to place anyway

Reply:
Ok i'll add that.